amp/TEMPLATE/reampv2/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "create-react-app",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://amp-haiti-future-v3-5-tc9.ampsite.net",
+  "proxy": "http://localhost:8080",
```

Comment:
Is this valid?

Reply:
actually neither of them is valid. I need to change it to ssc pr